src/components/tabs/FunctionsTab.vue

```diff
@@ -82,10 +136,49 @@ export default class FunctionsTab extends Vue {
     );
     if (name === null) return;
 
-    this.addFile({ filename: name, functions: [] });
+    this.addFile({ filename: `${name}.py`, functions: [] });
   }
+
+  private confirmClick = () => {
+    console.log('confirm');
+  };
+
+  private cancelClick = () => {
+    console.log('cancel');
+  };
 }
 </script>
 
 <style scoped>
+  #left {
+    width: 40%;
+    border-right: var(--grey) 1px solid;
+  }
+
+  #right {
+    width: 60%;
+    padding-top: 0;
+  }
+
+  .panel {
+    user-select: none;
+    padding: 1em;
+    border-top: var(--grey) 1px solid;
+  }
+
+  .button-list {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    height: calc(100% - 3.5em);
```

Comment:
I think it could be nice to have the buttons at the bottom to be at exactly the same vertical position as the ones in the IDE tab so that it doesn't look too weird when you switch between tabs (can change either those ones or the IDE ones). What do you think?

Reply:
We're going to change the IDE ones pretty soon, so I think it's fine